merkle/simple_map_test.go

```diff
@@ -0,0 +1,53 @@
+package merkle
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type strHasher string
+
+func (str strHasher) Hash() []byte {
+	return SimpleHashFromBytes([]byte(str))
```

Comment:
Any reason SimpleHashFromBytes disappeared?
https://github.com/tendermint/tmlibs/blob/50ad19541d61a60bdd6be6f407af3ed7abd6ba44/merkle/simple_tree.go#L62-L68

Reply:
meh - trying to simplify the exposed interface and it got replaced with `tmhash.Sum`

Reply:
OK, makes sense